src/recipes/screens/RecipeList.tsx

```diff
@@ -20,22 +29,32 @@ export const RecipeList = () => {
           <Text style={styles.errorText}>No meal available</Text>
         ) : (
           items.map(meal => (
-            <RecipeItem
-              name={meal.strMeal}
-              category={meal.strCategory}
-              imageUrl={meal.strMealThumb}
+            <TouchableOpacity
               key={meal.idMeal}
-            />
+              onPress={() => navigation.navigate('Details', { meal: meal })}>
```

Comment:
Put this function in a `useCallback`
https://fr.reactjs.org/docs/hooks-reference.html#usecallback